### ISSUE TITLE
[Feature Request] Sort output by frequency

### ISSUE THREAD

**Issue by gajewsk2** [HelenFoster/KanjiVocab#9]
Per discussion in https://github.com/HelenFoster/KanjiVocab/issues/8, it seems like it would be more effective studying to see the most frequent words first and by due date secondarily in the KanjiVocab fields (even if the card is unseen - eg morphman setting up the most relevant cards for you next).

**Comment by HelenFoster** [comment]
I'm prioritising first by mature > known > inactive > unscanned, and intending to keep that.

Within that, it's pretty much going by frequency in news. I can see that frequency in your cards might be more relevant. Probably "number of notes containing this word".

I'm not so sure about the due date. I can kind of see it for the unseen cards.

**Comment by gajewsk2** [comment]
Yeah that makes sense. I was thinking the frequency should be based off of the Japanese Vocab Frequency add-on, but I don't know that it actually matters which data set in my case.

**Comment by HelenFoster** [comment]
Hmm, I wonder if "number of notes containing this word" would actually be any use. In my deck, it would probably just mean I used the same example sentence for more than one word.
